diff --git a/pyproject.toml b/pyproject.toml
index 5a5295a..0dcb4cf 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "datamodel-code-generator"
-version = "0.0.0"
+version = "0.20.0"
 description = "Datamodel Code Generator"
 authors = ["Koudai Aono <koxudaxi@gmail.com>"]
 readme = "README.md"
@@ -22,20 +22,9 @@ classifiers = [
         "Programming Language :: Python :: Implementation :: CPython"]
 
 [build-system]
-requires = ["poetry_core>=1.0.0", "poetry-dynamic-versioning"]
+requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.poetry-dynamic-versioning]
-enable = true
-vcs = "git"
-# language=RegExp
-pattern = '^(?P<base>\d+\.\d+\.\d+)(-?((?P<stage>[a-zA-Z]+)\.?(?P<revision>\d+)?))?$'
-
-[tool.poetry-dynamic-versioning.substitution]
-files = ["*/version.py"]
-patterns = ["(^version: str = ')[^']*(')"]
-
-
 [tool.poetry.scripts]
 datamodel-codegen = "datamodel_code_generator.__main__:main"
 
